pkg/tests/ossm/smoke_test.go

```diff
@@ -96,13 +96,12 @@ func TestSmoke(t *testing.T) {
 			assertTrafficFlowsThroughProxy(t, ns)
 		})
 
-		// This test case is flaky and disabled for now: https://issues.redhat.com/browse/OSSM-4064
-		// t.NewSubTest(fmt.Sprintf("delete smcp %s", toVersion)).Run(func(t TestHelper) {
-		// 	t.Logf("This test checks whether SMCP %s deletion delete all the resources", env.GetSMCPVersion())
+		t.NewSubTest(fmt.Sprintf("delete smcp %s", toVersion)).Run(func(t TestHelper) {
+			t.Logf("This test checks whether SMCP %s deletion delete all the resources", env.GetSMCPVersion())
 
-		// 	t.LogStep("Delete SMCP and verify if this deletes all resources")
-		// 	assertUninstallDeletesAllResources(t, env.GetSMCPVersion())
-		// })
+			t.LogStep("Delete SMCP and verify if this deletes all resources")
+			assertUninstallDeletesAllResources(t, env.GetSMCPVersion())
```

Comment:
Why not inline this function? What's the benefit of having a single function call in a subtest? Yes, it does explain what the subtest does, but this is already explained in the log statements.

Reply:
Also, there are now two LogSteps one after the other (the one in line 102 and the one in line 186).

Reply:
Done